keras_cv/layers/preprocessing/with_mixed_precision_test.py

```diff
@@ -155,16 +185,56 @@ def test_can_run_in_mixed_precision(self, layer_cls, init_args):
                     "operations `tf.image.adjust_saturation`, and "
                     "`tf.image.adjust_hue`. Skipping."
                 )
+            if layer_cls in NO_BOUNDING_BOXES_TESTS:
+                self.skipTest(
+                    "There is currently no support for bounding boxes."
+                    "Skipping."              
+                )
 
         keras.mixed_precision.set_global_policy("mixed_float16")
 
         img = tf.random.uniform(
             shape=(3, 512, 512, 3), minval=0, maxval=255, dtype=tf.float32
         )
-        labels = tf.ones((3,), dtype=tf.float32)
-        inputs = {"images": img, "labels": labels}
 
+        bounding_boxes = {
+            "boxes": tf.convert_to_tensor(
+                [
+                    [
+                        [200, 200, 400, 400],
+                        [250, 250, 450, 450],
+                        [300, 300, 500, 500],
+                    ],  # Bounding boxes for image 1
+                    [
+                        [100, 100, 300, 300],
+                        [150, 150, 350, 350],
+                        [200, 200, 400, 400],
+                    ],  # Bounding boxes for image 2
+                    [
+                        [300, 300, 500, 500],
+                        [350, 350, 550, 550],
+                        [400, 400, 600, 600],
+                    ],
+                ],  # Bounding boxes for image 3
+                dtype=tf.float32,
+            ),
+            "classes": tf.ones((3, 3), dtype=tf.float32),
+        }
+
+        # inputs = {"images": img, "labels": labels}
+        inputs = {"images": img, "bounding_boxes": bounding_boxes}
+
+        # img = tf.random.uniform(
+        #     shape=(3, 512, 512, 3), minval=0, maxval=255, dtype=tf.float32
+        # )
+        # labels = tf.ones((3,), dtype=tf.float32)
+        # inputs = {"images": img, "labels": labels}
+        
         layer = layer_cls(**init_args)
+
+        if layer_cls == layers.CutMix:
```

Comment:
CutMix doesn't support bounding boxes, so this should just be included in `NO_BOUNDING_BOX_TESTS`

Reply:
Oh right!